Sources/SolanaSwift/APIClient/APIClient.swift

```diff
@@ -304,5 +304,13 @@ public protocol SolanaAPIClient {
     /// - Parameter requests: the requests
     /// - Returns: the result of mutiple requests
     func batchRequest(with requests: [JSONRPCRequestEncoder.RequestType]) async throws -> [AnyResponse<JSONRPCRequestEncoder.RequestType.Entity>]
+   
+    /// Perform a multiple same returning type requests at once
+    ///
+    /// - Experiment: Will be changed in future.
+    /// - Parameter method: method name
+    /// - Parameter params: params
+    /// - Returns: the result of mutiple requests
+    func batchRequest<Entity: Decodable>(method: String, params: [[Encodable]]) async throws -> [Entity?]
```

Comment:
Shouldn't return Entity, RPCClient request method must return Response instance. There are Error type of response you can use to filter response array